google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -2314,36 +2348,60 @@ Blob create(
    *   <li>The default credentials, if no credentials were passed to {@link StorageOptions}
    * </ol>
    *
-   * <p>Example of creating a signed URL that is valid for 2 weeks, using the default credentials
-   * for signing the URL.
+   * <p>Example of creating a signed URL that is valid for 1 week, using the default credentials for
+   * signing the URL, the default signing method (V2), and the default URL style (path-style):
    *
    * <pre>{@code
-   * String bucketName = "my_unique_bucket";
-   * String blobName = "my_blob_name";
-   * URL signedUrl = storage.signUrl(BlobInfo.newBuilder(bucketName, blobName).build(), 14,
-   *     TimeUnit.DAYS);
+   * String bucketName = "my-unique-bucket";
+   * String blobName = "my-blob-name";
+   * URL signedUrl = storage.signUrl(
+   *     BlobInfo.newBuilder(bucketName, blobName).build(),
+   *     7, TimeUnit.DAYS);
    * }</pre>
    *
    * <p>Example of creating a signed URL passing the {@link SignUrlOption#withV4Signature()} option,
-   * which enables V4 signing.
+   * which enables V4 signing:
+   *
+   * <pre>{@code
+   * String bucketName = "my-unique-bucket";
+   * String blobName = "my-blob-name";
+   * URL signedUrl = storage.signUrl(
+   *     BlobInfo.newBuilder(bucketName, blobName).build(),
+   *     7, TimeUnit.DAYS,
+   *     Storage.SignUrlOption.withV4Signature());
+   * }</pre>
+   *
+   * <p>Example of creating a signed URL passing the {@link SignUrlOption#withVirtualHostedStyle()}
+   * option, which specifies the bucket name in the hostname of the URI, rather than in the path:
+   *
+   * <pre>{@code
+   * URL signedUrl = storage.signUrl(
+   *     BlobInfo.newBuilder(bucketName, blobName).build(),
+   *     1, TimeUnit.DAYS,
+   *     Storage.SignUrlOption.withVirtualHostedStyle());
+   * }</pre>
```

Comment:
Cold you provide an example of a result so people could distinguish easier?

Reply:
I added example URLs in the other docstrings (for `withPathStyle()` and `withVirtualHostedStyle()`.  It seems like the specifics belong there, rather than in the `signUrl()` docstring.  Thoughts?